R/pkg/R/functions.R

```diff
@@ -426,7 +426,7 @@ setMethod("covar_pop", signature(col1 = "characterOrColumn", col2 = "characterOr
 
 #' cos
 #'
-#' Computes the cosine of the given value.
+#' Computes the cosine of the given value. Units in radians.
```

Comment:
BTW, just while you are here @felixcheung, does this change itself look okay to you?